### ISSUE TITLE
Wrong Districts in 20161108__oh__general__precinct

### ISSUE THREAD

**Issue by hodgesmr** [openelections/openelections-data-oh#59]
There are incorrect districts in [20161108__oh__general__precinct](https://github.com/openelections/openelections-data-oh/blob/master/2016/20161108__oh__general__precinct.csv).

I'm not sure if it's incorrect across the board, but I discovered it in the case of OH-01 and OH-02 both crossing into Hamilton County.

For example:
```
Hamilton,BLUE ASH 1-A,AAA,Representative to Congress,1,R,Steve Chabot,0
Hamilton,BLUE ASH 1-A,AAA,Representative to Congress,1,,Sholom D. Keller,0
Hamilton,BLUE ASH 1-A,AAA,Representative to Congress,1,,Kiumars G. Kiani,0
Hamilton,BLUE ASH 1-A,AAA,Representative to Congress,1,D,Michele Young,0

...

Hamilton,BLUE ASH 1-A,AAA,Representative to Congress,2,,Janet Everhard,0
Hamilton,BLUE ASH 1-A,AAA,Representative to Congress,2,D,William R. Smith,157
Hamilton,BLUE ASH 1-A,AAA,Representative to Congress,2,R,Brad Wenstrup,364
```

I do not believe the `Hamilton,BLUE ASH 1-A,AAA` precinct should include any District 1 candidates.

I checked the 2016 Hamilton Country source file [from the sources repository](https://github.com/openelections/openelections-sources-oh/blob/master/2016/2016%20HAMILTON%20OH.xlsx), and the aforementioned Blueash district appears on the District 2 sheet, but not in the District 1 sheet.

I also grabbed an [export from the Hamilton County board of elections](http://results.enr.clarityelections.com/OH/Hamilton/64172/184023/Web01/en/summary.html#) and confirmed that the precincts don't accommodate multiple districts.

I found that **all** precincts in Hamilton County in `20161108__oh__general__precinct` appear for District 1 and District 2 candidates.

It's entirely possible that this issue extends beyond Hamilton County, but I haven't yet had a chance to investigate.

**Comment by dwillis** [comment]
Am I correct in thinking that precincts with all candidates receiving 0 votes is the pattern here?

**Comment by hodgesmr** [comment]
That's what I'm currently investigating. I've limited my scope to only the `Representative to Congress` office for now.

I've found many more instances that do follow this pattern beyond just Hamilton County.

The good news is, with that scope and pattern in mind, I'm not finding any instances where a given precinct has 0 votes for _all_ candidates.

The biggest unknown here is: Are there any precincts that appropriately service more than one district?

I might have a PR addressing this, if I can be confident in that pattern.

**Comment by dwillis** [comment]
I suspect that it might require us to look at state legislative races to confirm whether split precincts exist.

**Comment by hodgesmr** [comment]
Here's an example of a precinct that has non-zero votes for two different congressional districts:

```
Huron,HARTLAND TWP,ABS,Representative to Congress,4,D,Janet Garrett,1
Huron,HARTLAND TWP,ABS,Representative to Congress,4,R,Jim Jordan,3
Huron,HARTLAND TWP,ABS,Representative to Congress,7,R,Bob Gibbs,301
Huron,HARTLAND TWP,ABS,Representative to Congress,7,I,Dan Phillip,30
Huron,HARTLAND TWP,ABS,Representative to Congress,7,D,Roy Rich,104
```

Looking at the [source pdf for Huron County](https://github.com/openelections/openelections-sources-oh/blob/master/2016/2016%20HURON%2011-8-16-precinct-by-precinct-official-results1.pdf), this appears to be correct.

<img width="796" alt="screen shot 2018-03-24 at 11 54 07 am" src="https://user-images.githubusercontent.com/1208575/37867202-6dd5c16e-2f5a-11e8-9d19-49e86290f7ef.png">
<img width="686" alt="screen shot 2018-03-24 at 11 55 12 am" src="https://user-images.githubusercontent.com/1208575/37867206-7489976a-2f5a-11e8-86e2-0bb683f43836.png">

Since it's apparently possible for a precinct to service two different congressional districts, I don't think we can confidently say that the pattern of zero-votes is adequate. For example, a precinct could correctly service District A and District B _and_ actually have recorded zero votes for the candidates in District B. In that scenario, we wouldn't want to filter out District B's zero-vote results from the precinct results.

**Comment by dwillis** [comment]
Yeah, split precincts are definitely a thing, and we should document this in general.

**Comment by hodgesmr** [comment]
For the sake of proving to myself that my hypothetical my last comment exists:

```
Franklin,DUB 1-B,ARH,Representative to Congress,3,R,John Adams,0
Franklin,DUB 1-B,ARH,Representative to Congress,3,D,Joyce Beatty,0
Franklin,DUB 1-B,ARH,Representative to Congress,12,D,Ed Albertson,111
Franklin,DUB 1-B,ARH,Representative to Congress,12,,John J. Baumeister,0
Franklin,DUB 1-B,ARH,Representative to Congress,12,G,Joe Manchik,9
Franklin,DUB 1-B,ARH,Representative to Congress,12,R,Pat Tiberi,351
Franklin,DUB 1-B,ARH,Representative to Congress,15,R,Steve Stivers,0
Franklin,DUB 1-B,ARH,Representative to Congress,15,D,Scott Wharton,0
```

and from the [source file](https://github.com/openelections/openelections-sources-oh/blob/master/2016/2016%20franklin%20oh.xlsx):

<img width="719" alt="screen shot 2018-03-24 at 12 49 33 pm" src="https://user-images.githubusercontent.com/1208575/37867752-3b3397a6-2f62-11e8-89aa-8c2fcb964ebb.png">

**However**: Franklin County's source document doesn't leave any of the vote fields blank. It looks like they `0` every field, even if a precinct is not part of a given district (for whichever office you're looking at).
